packages/math-input/src/types.ts

```diff
@@ -9,6 +9,14 @@ import {
 
 import type {CursorContext} from "./components/input/cursor-contexts";
 
+export interface MathQuillInterface {
+    write: (input: string) => void;
+    cmd: (input: string) => void;
+    keystroke: (input: string) => void;
+    typedText: (input: string) => void;
+    latex: () => string;
```

Comment:
Is it worth a short comment on each of these describing what they do/represent? 

I'm assuming the first 4 a functions to write stuff into the math quill input and the last extracts the current state in TeX format.... is that correct?